cmd/namespace.go

```diff
@@ -118,17 +147,32 @@ To change the namespace, you will have to contact CircleCI customer support.
 
 	confirm := fmt.Sprintf("Are you sure you wish to create the namespace: `%s`", namespaceName)
 	if opts.noPrompt || opts.tty.askUserToConfirm(confirm) {
-		_, err := api.CreateNamespace(opts.cl, namespaceName, opts.args[2], strings.ToUpper(opts.args[1]))
-
+		_, err = api.CreateNamespace(opts.cl, namespaceName, opts.args[2], strings.ToUpper(opts.args[1]))
 		if err != nil {
 			return err
 		}
 
 		fmt.Printf("Namespace `%s` created.\n", namespaceName)
 		fmt.Println("Please note that any orbs you publish in this namespace are open orbs and are world-readable.")
 	}
+	return err
+}
 
-	return nil
+func createNamespace(opts namespaceOptions) error {
+	var err error
+	namespaceName := opts.args[0]
```

Comment:
same, this will throw if no args passed. in line 79, we can use a range to force at least 1 arg `Args: cobra.RangeArgs(1, 3),` instead of `Args: cobra.MaximumNArgs(3),`